FontLoader.js

```diff
@@ -661,8 +745,8 @@
 					return;
 				}
 			}
-			
-			if (!this._continuous) {
+
+			if (!this._continuous || this._delegate.isFinished()) {
```

Comment:
This code was added in our version of the fontloader some time ago. We saw that there are cases where `prepareForWatch` was called after the element was removed from the DOM, resulting in errors.

To prevent this, I moved the check a bit to the top. This is in line with what you are doing with `if (this._state !== SizeWatcher.states.watchingForSizeChange)`.

It would be great if we could add this to a test to make sure the code handles all kinds of race conditions on the right way (same for the other things you filtered out by checking my code). Any thoughts on that? Would you want me to add this case to the compat-tests?